ksonnet-util/kausal.libsonnet

```
@@ -261,7 +261,11 @@ k {
         local serviceAccount = $.core.v1.serviceAccount,
 
         service_account:
-          serviceAccount.new(name),
+          serviceAccount.new(name) +
+          if (pullSecrets == {}) then {}
+          else serviceAccount.withImagePullSecrets(pullSecrets) +
+               if (annotations == {}) then {}
+               else serviceAccount.metadata.withAnnotationsMixin(annotations),
```

You can wrap the if/else in brackets to have preserve the indentation.
```suggestion
          serviceAccount.new(name)
          + (
            if (pullSecrets == {}) then {}
            else serviceAccount.withImagePullSecrets(pullSecrets)
          ) + (
            if (annotations == {}) then {}
            else serviceAccount.metadata.withAnnotationsMixin(annotations)
          ),
```

---

Thanks for the feedback!  Updated per your suggestion.